build/template-build-ios-automation.yaml

```diff
@@ -12,33 +12,80 @@ steps:
     secureFile: AppCenteriOSBuildCert.p12
 
 - task: DownloadSecureFile@1
-  displayName: 'Download Provisioning Provile'
+  displayName: 'Download Provisioning Profile'
   inputs:
     secureFile: 'TestFile.mobileprovision'
 
+- task: UseDotNet@2
+  displayName: 'Install .NET 6 SDK'
+  inputs:
+    version: 6.0.x
+    performMultiLevelLookup: true
+
+- task: UseDotNet@2
+  displayName: Use .NET 6.0
+  inputs:
+    packageType: 'sdk'
+    version: '6.0.x'
+
+- task: Bash@3
+  displayName: Install MAUI
+  inputs:
+    targetType: 'inline'
+    script: |
+      dotnet nuget locals all --clear 
+      dotnet workload install maui --source https://aka.ms/dotnet6/nuget/index.json --source https://api.nuget.org/v3/index.json
+      dotnet workload install android ios maccatalyst tvos macos maui wasm-tools --source https://aka.ms/dotnet6/nuget/index.json --source https://api.nuget.org/v3/index.json
+
 - task: NuGetCommand@2
   displayName: 'NuGet restore'
   inputs:
     restoreSolution: LibsMacOS.sln
 
-- task: XamariniOS@1
-  displayName: 'Build Xamarin.iOS solution LibsMacOS.sln'
+- task: Bash@3
+  displayName: List SDKs
   inputs:
-    solutionFile: LibsMacOS.sln
-    configuration: Debug
-    clean: true
-    runNugetRestore: false
-    args: '/p:APPCENTER_BUILD=1 /p:RunCodeAnalysis=false'
-    buildToolOption: msbuild
-    signingP12File: '$(Agent.TempDirectory)/AppCenteriOSBuildCert.p12'
-    signingP12Password: '$(AppCenterDotNetiOSTestCertPassword)'
-    signingProvisioningProfileFile: '$(Agent.TempDirectory)/TestFile.mobileprovision'
+    targetType: 'inline'
+    script: |
+      dotnet --list-sdks 
+
+# https://docs.microsoft.com/en-us/dotnet/maui/macos/deployment/overview
+- task: Bash@3
+  displayName: Build the Lib
+  inputs:
+    targetType: 'inline'
+    script: |
+      cd src/client/Microsoft.Identity.Client
+      dotnet build -f net6.0-maccatalyst -c Release -o ./artifacts
+
+- task: Bash@3
+  displayName: Build the Basic App
+  inputs:
+    targetType: 'inline'
+    script: |
+      pwd
+      cd tests/devapps/MauiApps/MauiAppBasic
+      dotnet build -f net6.0-maccatalyst -c Release -o ./artifacts
+      dotnet publish -f net6.0-maccatalyst -c Release -p:BuildIpa=True -o ./artifacts
+
+# - task: XamariniOS@1
```

Comment:
do we need this?

Reply:
Kept temporarily if we decide to create Xamarin.Forms app for App Center testing.